JobConfig/pileup/epilog.fcl

```diff
@@ -1 +1,12 @@
-
+# don't need to keep StepPointMC info
+physics.producers.compressDetStepMCs.stepPointMCTags : [ ]
+# remove some intermediate genealogy steps
+physics.producers.compressDetStepMCs.compressionOptions : @local::DetStepCompression.extraCompression
+# no MC Trajectories
+physics.producers.compressDetStepMCs.mcTrajectoryTag : ""
+# don't create MCTrajectories
+physics.producers.g4run.TrajectoryControl.produce : false
+# only detector SDs
+physics.producers.g4run.SDConfig.enableSD: [@sequence::physics.producers.g4run.SDConfig.enableSD ]
+# no special deltas treatment for pileup
+physics.producers.StrawGasStepMaker.KeepDeltasModule : ""
```

Comment:
A lot of epilog settings in JobConfig/pileup/epilog.fcl
It looks like most files in JobConfig/pileup/ are obsolete?
I am thinking of deuteron.fcl, dio.fcl, etc.   Perhaps they should be removed, then all
the setting can go into a single remaining Pileup.fcl?

Reply:
I agree the active content of pileup is small, but the content can't be reduced to a single top-level script as the config for pileup detectorsteps is reused in beam (resamplers). 
epilog is the only way to amend config without nesting the prolog (which Pasha objected to) or duplicating blocks of common config, which is error-prone and a maintenance burden/error source.
I left the dedicated single-particle production scripts in case someone wanted to use them to study those particles as calibration sources, but they don't belong in pileup.  I'll move or remove them.